createdisk.sh

```diff
@@ -114,6 +114,7 @@ After=sys-devices-virtual-net-%i.device
 Restart=on-failure
 Environment="GV_VSOCK_PORT=1024"
 EnvironmentFile=-/etc/sysconfig/gv-user-network
+ExecCondition=/usr/local/bin/crc-check-cloud-env.sh
```

Comment:
we need to do the reverse, the `crc-check-cloud-env.sh` script returns success (exit code 0) when its running in the cloud, so this makes it run in the cloud and not run when running on local virtualization

this should be `ExecCondition=/bin/bash -c '! /usr/local/bin/crc-check-cloud-env.sh'`